quandl/utils/api_key_util.py

```diff
@@ -7,3 +9,9 @@ def init_api_key_from_args(params):
         # set api key
         if 'api_key' in params:
             ApiConfig.api_key = params.pop('api_key')
+        # issue warning if api was previously set with params
+        # and now is not
+        elif (ApiConfig.api_key is not None) and os.isatty(0):
```

Comment:
Will this trigger every time api key is not set whether or not it had been in the past?